### PR TITLE
Update dates on FAQ page

### PR DESCRIPTION
**Pull request by akshitac8**
Changed Location on [FAQ Page](https://in.pycon.org/2018/faq.html) in reference with the issue #42

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -139,17 +139,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
         <div class="pycon-faq">
           <h5><b>Q. What are the workshop, Developer Sprints and conference timings?</b></h5>
           <ul class="faq-answer">
-              <li> - Workshop/Developers Sprints days (Aug 25 and Aug 26, 2018): 08:00 AM - 06:30 PM.</li>
-              <li>- Conference days (Aug 22 to Aug 24, 2018): 08:00 AM </li>
+              <li> - Workshop/Developers Sprints days (Oct 8 and Oct 9, 2018): 08:00 AM - 06:30 PM.</li>
```
Comment (ananyo2012) [inline faq.html]: Workshops - Oct 5
Dev Sprints - Oct 8 and Oct 9
Move it in 2 separate lists

File: faq.html [hunk 2]
```diff
@@ -139,17 +139,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
...
           </ul>
         </div> 
         <div class="pycon-faq">
-          <h5><b>Q. What all does the Conference ticket (Aug 22 to Aug 24, 2018) include? </b></h5>
+          <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li> - Full conference pass for three days.</li>
                 <li>- Breakfast, Lunch and Hi-Te</li>
-                <li>- PyCon India T-shirt which will be distributed on Aug 25, 2018.</li>
+                <li>- PyCon India T-shirt which will be distributed on Oct 9, 2018.</li>
```
Comment (ananyo2012) [inline faq.html]: This will be distributed probably in Oct 6. @chirag200666 Should we keep this or remove this point for now ?

File: faq.html [hunk 3]
```diff
@@ -139,17 +139,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
...
+              <li>- Conference days (Oct 6 and Oct 7, 2018): 08:00 AM </li>
               <li>- 5.30 PM.</li>
           </ul>
         </div> 
         <div class="pycon-faq">
-          <h5><b>Q. What all does the Conference ticket (Aug 22 to Aug 24, 2018) include? </b></h5>
+          <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li> - Full conference pass for three days.</li>
                 <li>- Breakfast, Lunch and Hi-Te</li>
```
Comment (ananyo2012) [inline faq.html]: Can you correct this spelling error ?

File: faq.html [hunk 4]
```diff
@@ -139,17 +139,17 @@ <h5><b>Q. Will the Internet and WiFi setup be good enough for a large crowd? Sho
...
+              <li> - Workshop/Developers Sprints days (Oct 8 and Oct 9, 2018): 08:00 AM - 06:30 PM.</li>
+              <li>- Conference days (Oct 6 and Oct 7, 2018): 08:00 AM </li>
               <li>- 5.30 PM.</li>
           </ul>
         </div> 
         <div class="pycon-faq">
-          <h5><b>Q. What all does the Conference ticket (Aug 22 to Aug 24, 2018) include? </b></h5>
+          <h5><b>Q. What all does the Conference ticket (Oct 6 and Oct 7, 2018) include? </b></h5>
           <ul class="faq-answer">
                 <li> - Full conference pass for three days.</li>
```
Comment (ananyo2012) [inline faq.html]: Conference ticket is for 2 days (Oct 6 - 7).
For Workshop and Dev Sprints there will be separate ticket. Can you update this as well ?

**Comment by ananyo2012** [comment]
@akshitac8 Thanks for your contribution to the PyCon India 2018 website. It will be great if you can make the changes as suggested. The overall schedule finalized is
```
Oct 5 - Workshops
Oct 6-7 - Conference
Oct 8-9 - Dev Sprints
```

**Comment by akshitac8** [comment]
Done :slightly_smiling_face:

File: faq.html (hunk 4 above)
Comment (realslimshanky) [reply faq.html]: @akshitac8 the emphasis here is on `three days` which needs to be changed to `two days`.

**Comment by realslimshanky** [comment]
@akshitac8 please look into the review comments. There are a couple of issues left to fix.

**Comment by akshitac8** [comment]
Done :slightly_smiling_face:

**Comment by realslimshanky** [comment]
@akshitac8  Please also look into line 151 for the typo `Hi-Te` -> `Hi-Tea`

**Comment by akshitac8** [comment]
Thank you :slightly_smiling_face:

**Comment by akshitac8** [comment]
Are there any further changes to be done?

**Comment by ananyo2012** [comment]
@akshitac8 Merged in your changes.

**Pull request merged**